src/CapabilityService.WebApi/Enablers/PrometheusHealthCheck/Middleware.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+
+namespace DFDS.CapabilityService.WebApi.Enablers.PrometheusHealthCheck
+{
+    public static class Middleware
+    {
+        public static IApplicationBuilder UsePrometheusHealthCheck(
+            this IApplicationBuilder app
+        )
+        {
+            app.UseHealthChecks("/healthz", new HealthCheckOptions
```

Comment:
Cant we get into an error state with this Enabler if for some reason .AddHealthChecks() is removed from StartUp.cs? Maybe we should create an extension method for prometheus to call .AddHealthChecks()?

Reply:
I tested it and we will get a error if AddHealthChecks is not done on services.

I have added a dependency check before starting the  PrometheusHealthCheck.

Reply:
I think thats an excellent solution if we do not foresee a future where the Prometheus-enabler will have need of other dependencies. However it might be better to implment an AddPrometheusHealthCheck extension method to just call the AddHealthCheck so we stay consistent with other Enablers?

Reply:
`AddPrometheusHealthCheck have been added to code`